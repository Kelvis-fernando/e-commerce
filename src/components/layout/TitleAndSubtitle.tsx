import { TitleAndSubtitleContainer } from "../../styles/layout/titleAndSubtitle";
import { TitleAndSubtitleProps } from "../../types/titleAndSubtitle";

const TitleAndSubtitle = ({ title, subtitle }: TitleAndSubtitleProps) => {
  return (
    <TitleAndSubtitleContainer>
      <h1>{title}</h1>
      <h3>{subtitle}</h3>
    </TitleAndSubtitleContainer>
  );
};

export default TitleAndSubtitle;
