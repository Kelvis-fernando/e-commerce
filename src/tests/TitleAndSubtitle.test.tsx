import React from "react";
import { render } from "@testing-library/react";
import TitleAndSubtitle from "../components/layout/TitleAndSubtitle";

describe("TitleAndSubtitle", () => {
  test("should render and match snapshot", () => {
    const { container } = render(<TitleAndSubtitle title={""} />);

    expect(container).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
