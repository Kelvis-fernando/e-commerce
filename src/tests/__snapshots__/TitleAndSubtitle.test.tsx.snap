// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleAndSubtitle should render and match snapshot 1`] = `
<div>
  <div
    class="sc-bcXHqe tMTwO"
  >
    <h1 />
    <h3 />
  </div>
</div>
`;
